charts/backstage/values.yaml

```diff
@@ -14,6 +17,10 @@ upstream:
       repository: janus-idp/backstage-showcase
       tag: latest
     command: []
+    args:
+      - '--config'
+      - dynamic-plugins-root/app-config.dynamic-plugins.yaml
```

Comment:
Can you use `upstream.backstage.extraAppConfig` value instead?

https://github.com/backstage/charts/tree/main/charts/backstage#pass-extra-configuration-files

Reply:
I don't think it is possible, since this additional app-config file is generated by the installation script in the init container, based on the content of the `dynamic-plugins.yaml` file (provided through the `dynamic-plugins`  config map, but originally defined in the `global.dynamicPlugins` Helm value.

Reply:
@davidfestal I don't think this method would work since these new arguments are not applied unless you also change the `command` line from `command: []` to `command: ['node', 'packages/backend']` to override the [entrypoint](https://github.com/janus-idp/backstage-showcase/blob/0d27cbbda96f6ae2bde21bb1540a3e326b84ac18/docker/Dockerfile#L138) defined when building the container image. 

However, if you do just add `command: ['node', 'packages/backend']`  alongside the edit you made above, it won't apply the `app-config.yaml` and other config files that are normally applied by default, which can cause backend to fail due to missing configurations.

Reply:
I don't think we need to override the `command`. Kubernetes starts the pod via `[ENTRYPOINT] [CMD]` which translates to `[command] [args]`, so this solution should work fine.